EDI/Operation/04_ManageLineItemsFileStt.md

```diff
@@ -0,0 +1,185 @@
+
+2. Click the *File settings* tab.   
+    The *File settings* tab is displayed.
+
+     ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefEmptyfileSettings.png "[File settings tab]")
+
+3. Click the *Release* drop down list and select the required EDIFACT release. In our example, it is **D07a**.
+
+4. Click the *Message type* drop-down list and select the required EDIFACT message type. In our example, it is **INVOIC**. You can use the search function on top of the drop-down list.
+
+5. Enter the *Reference node path*. Use the following syntax:    
+    /M_[Message type, for example INVOIC]\_[EDIFACT version number (last three digits)]/ G\_[Message type, for example INVOIC]\_[EDIFACT version number (last three digits)]\_SG[Group number].     
+    For example: /M_INVOIC_07A/G_INVOIC_07A_SG26   
+    Alternatively, you can copy the two parts from the header file message structure.  
+
+    >[Info] Note that the line items (LIN segment) are included in separate groups, such as the SG26 or SG27 group for the INVOIC.
+
+    ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefMessagData.png)
+
+ 
+6. Click the ![Get](../../Assets/Icons/Download.png "[Get]") [GET] button.   
+    The message structure of the SG26/SG27 group is displayed. The segment is highlighted in red because it is still empty.    
+    It starts with the SG26/SG27 element. 
+
+    ![Message](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructureLineItems.png "[Message]")
+
+
+7. If desired, expand the message structure by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button.   
+    The message structure with its single segments is displayed. It starts with the S_LIN segment. The ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button has changed to an ![Expanded](../../Assets/Icons/NodeExpanded.png "[Expanded]") (Expanded) button.
+
+    ![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructureSG26.png "[Message structure]")
+
+8. Click the [SAVE] button in the top right corner.   
+     - The *Exporter definitions* view is displayed.   
+     - If you now open the definition again for editing, both the *Data Sources* tab and the *Mapping settings* tab is additionally displayed. 
+     - In the header definition file settings, you are no longer able to expand the *G_INVOIC_07A_SG26* or *G_INVOIC_07A_SG27* group.
+
+9. If desired, create the data source and define the mapping setting. For detailed information, see [Prepare definition of file settings](./01_ManageDefinitions.md#prepare-definition-of-file-settings).
+
+
+
+## Specify line item number
+
+Start specifying the line item segments by filling out the single elements of the S_LIN (Line item) segment. This procedure shows how to edit the line item number on the invoice, so that each invoice line item gets a unique and ascending number.
```

Comment:
```suggestion
Start specifying the line item segments by filling out the single elements of the S_LIN (line item) segment. This procedure shows how to edit the line item number on the invoice, so that each invoice line item gets a unique and ascending number.
```

Reply:
done